apps/src/responsive.jsx

```diff
@@ -0,0 +1,119 @@
+/* A very simple responsive layout system.
+ */
+
+import * as utils from './utils';
+import $ from 'jquery';
+
+export default class Responsive {
+
+  constructor(responsiveWindowWidth) {
+    this.responsiveWindowWidth = responsiveWindowWidth;
+  }
+
+  /**
+   * Gets the container width.
+   * Returns either a number (e.g. 1170) or a string (e.g. "97%").
+   */
+  getResponsiveContainerWidth() {
+    const windowWidth = $(window).width();
+
+    if (windowWidth >= 970+50) {
+      return 970;
+    } else {
+      return "97%";
+      // return Math.round(0.97 * windowWidth);
+    }
+  }
+
+  // makeEnum comes from apps/src/utils
+  static ResponsiveSize = utils.makeEnum('lg', 'md', 'sm', 'xs');
```

Comment:
any reason not to do `import { makeEnum } from './utils'` instead of importing all of utils? Also, this comment seems unnecessary